project_extra/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============
+Project Extra
```

Comment:
@noahzaozao any reason why we add this module in the repo, especially in this PR? Is is because another module depends on this one? Else, I'd rather add this module in a separate PR.